tests/envs/TYPEKIT_API_KEY

```diff
@@ -0,0 +1 @@
+78ybi67812345niy8902345n9679123
```

Comment:
Yep, this is all good - it's a made up string, will be used for testing eventually.